package.json

```diff
@@ -23,7 +23,13 @@
     "@influxdata/influxdb-client-apis": "^1.35.0",
     "@pinia/nuxt": "^0.5.5",
     "@prisma/client": "^5.21.1",
+    "@turf/difference": "^7.1.0",
     "@turf/great-circle": "^7.1.0",
+    "@turf/helpers": "^7.1.0",
+    "@turf/intersect": "^7.1.0",
+    "@turf/meta": "^7.1.0",
+    "@turf/truncate": "^7.1.0",
+    "@turf/union": "^7.1.0",
```

Comment:
Please validate do you still use ALL of those added deps

Reply:
Yes, all 4 are in use.